app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -19,7 +19,7 @@ talk_to_us: false
 inset_text:
   applications-open:
     text: |-
-      <p>Applications for the <a href="https://get-a-teacher-relocation-payment.education.gov.uk">international relocation payment (IRP)</a> are open for teachers between 2 April and 16 June 2024.</p>
+      <p>Applications for the <a href="https://get-a-teacher-relocation-payment.education.gov.uk">international relocation payment (IRP)</a> are now closed. You can express your interest in the IRP by emailing us at IRP.ExpressInterest@education.gov.uk. We'll email you when applications reopen in autumn 2024.</p>
```

Comment:
@laurahermionetennant92 Should we remove the link here? There doesn't seem much point directly users to the IRP application page if applications are closed

cc @Sarah-DfE

Reply:
I think the advantage of keeping the link is that they can bookmark and check back? Otherwise it's very hard to find the application form.